Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -57,8 +57,72 @@ namespace ReactionRunnerUtils {
 //! returns whether or not all reactants matched
 namespace {
 const unsigned int MatchAll = UINT_MAX;
+
+class StereoBondEndCap {
```

Comment:
Some documentation about what this is or what it's for would be nice.

Reply:
Oops, I wrote a small note, but it must have gotten lost in one of the changes. I'll write something again.